includes/class-wc-siftscience-format.php

```diff
@@ -22,20 +22,20 @@
 	require_once( 'class-wc-siftscience-format-transaction.php' );
 
 	class WC_SiftScience_Format {
-		public $items;
-		public $login;
-		public $account;
-		public $order;
-		public $cart;
-		public $transactions;
+		public $_items;
```

Comment:
public props should not have underscore in the name

Reply:
i read it private sorry